src/StoreManager.ts

```diff
@@ -70,27 +70,10 @@ class StoreManager {
 
   private load = async (address: string): Promise<OrbitDBStore> => {
     log(`Opening store with address ${address}`);
-
     // I think this is done anyways by orbit, but just in case
     const pinHeadHash = (storeAddress: string, ipfsHash: string): void => {
       this.ipfsNode.pinHash(ipfsHash);
     };
-
-    const handlePeerExchanged = (
-      peer: string,
-      storeAddress: string,
-      heads: Entry[],
-    ): void => {
-      // @todo The `peer` argument sometimes is undefined. It might not be a big
-      // problem as the replication works properly. But we should keep an eye on
-      // it.
-      log(
-        `Store "${address}" replicated for ${peer}. Got ${
-          heads.length
-        } new heads.`,
-      );
-      this.events.emit('stores:replicated', { address, heads, peer });
```

Comment:
So this will be totally fire and forget from the client-side? No hook to store `replicated`?

Reply:
Is it as flaky as depending on `peer.exchanged`?

Reply:
Yes, but it was like that before. We didn’t use the event in the client, only in the tests